omod/src/main/resources/apps/ugandaemr_reports_app.json

```diff
@@ -583,6 +583,15 @@
         "order": 20,
         "requiredPrivilege": "App: ugandaemrreports.HMIS055-HCTRegister"
       },
+      {
+        "id": "HMIS 008-PNC Register",
+        "extensionPointId": "org.openmrs.module.ugandaemr.reports.registers2019",
+        "type": "link",
+        "label": "HMIS 008 : PNC Register",
+        "url": "reportingui/runReport.page?reportDefinition=ffed8404-7f81-4c9e-a9eb-db2673397cc0",
+        "order": 20,
+        "requiredPrivilege": "App: ugandaemrreports.HMIS008-PNCRegister"
```

Comment:
@solemabrothers where is this privilege defined?

Reply:
@ssmusoke many of these previllages in reports are place holders, we havent defined them yet.

Reply:
@solemabrothers if you do not define them then the reports cannot be displayed - each privilege has to be defined and added to the appropriate roles as part of the PR